android/app/build.gradle

```diff
@@ -256,8 +256,10 @@ dependencies {
         exclude group: 'com.soywiz.korlibs.krypto', module: 'krypto-jvm'
         exclude group: 'com.google.protobuf', module: 'protobuf-javalite'
         exclude group: 'io.ipfs.multibase.Base16'
+        exclude group: 'org.bouncycastle', module: 'bcprov-jdk15to18'
+        exclude group: 'org.jetbrains.kotlinx', module: 'kotlinx-coroutines-core'
     }
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
+    //implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.2")
```

Comment:
I had to uncomment the kotlinx-serialization-json lib, PrismModule wasn't compiling without it.  Can you double check this on your side?

Reply:
Everything else looks good!

Reply:
if I uncomment it, I receive "More than one file was found with OS independent path 'META-INF/kotlinx-serialization-json.kotlin_module'.". Weird.

Reply:
I tried 'clean' and 'cleanBuildCache' from my IDE, and then 'build'.  and still got the build error.

I also tried from the command line:
rm -Rf node_modules;yarn cache clean;yarn install
and
yarn android

and got the same build error.
Did you try those?